config.json

```diff
@@ -48,7 +48,8 @@
     "search_relevancy_threshold": 0.8,
     "data_formats": "all",
     "eval_data_jsonl_file_path": "./artifacts/eval_data.jsonl",
-    "chunking_strategy": "basic",
+    "chunking_strategy": "azure-document-intelligence",
```

Comment:
Changing the `chunking_strategy` should be discussed in the issue/agreed on in the Teams channel,
as for some cases the basic approach which is cheaper is good enough